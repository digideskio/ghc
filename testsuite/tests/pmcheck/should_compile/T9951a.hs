{-# OPTIONS_GHC -Wall        #-}
{-# LANGUAGE OverloadedLists #-}

module T9951a where

f :: [a] -> ()
f x = case x of
  [] -> ()
  (_:_) -> ()

