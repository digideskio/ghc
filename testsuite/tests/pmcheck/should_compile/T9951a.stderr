[1 of 1] Compiling T9951a           ( T9951a.hs, T9951a.o )
